pysatSeasons/occur_prob.py

```diff
@@ -61,38 +75,103 @@ def daily2D(inst, bin1, label1, bin2, label2, data_label, gate,
 
     """
 
-    return _occurrence2D(inst, bin1, label1, bin2, label2, data_label, gate,
-                         by_orbit=False, returnBins=returnBins)
+    return _occurrence2D(const, bin1, label1, bin2, label2, data_label, gate,
+                         by_orbit=False, returnBins=returnBins,
+                         return_bins=return_bins)
+
 
+def by_orbit2D(const, bin1, label1, bin2, label2, data_label, gate,
+               returnBins=None, return_bins=False):
+    """2D Occurrence Probability of `data_label` orbit-by-orbit over a season.
 
-def by_orbit2D(inst, bin1, label1, bin2, label2, data_label, gate,
-               returnBins=False):
-    """2D Occurrence Probability of data_label orbit-by-orbit over a season.
+    .. deprecated:: 0.2.0
+              `returnBins` will be removed in v0.3.0, it is replaced by
+              `return_bins` because it has standards compliant case.
 
-    If data_label is greater than gate atleast once per orbit, then a
+    If `data_label` is greater than `gate` at least once per orbit, then a
     100% occurrence probability results. Season delineated by the bounds
     attached to Instrument object.
-    Prob = (# of times with at least one hit)/(# of times in bin)
+    Probability = (# of times with at least one hit) / (# of times in bin)
 
     Parameters
     ----------
-    inst: pysat.Instrument
-        Instrument to use for calculating occurrence probability
-    binX: array-like
+    const : pysat.Instrument or pysat.Constellation
+        Instrument/Constellation to use for calculating occurrence probability.
+    bin1, bin2 : array-like
         List holding [min, max, number of bins] or array-like containing
-        bin edges, where X = 1, 2
-    labelX: string
-        identifies data product for binX, where X = 1, 2
-    data_label: list of strings
-        identifies data product(s) to calculate occurrence probability
-    gate: list of values
-        values that data_label must achieve to be counted as an occurrence
-    returnBins: Boolean
-        if True, return arrays with values of bin edges, useful for pcolor
+        bin edges.
+    label1, label2 : str
```

Comment:
Recommend separating into individual entries

Reply:
I followed the numpydoc standard on this. https://numpydoc.readthedocs.io/en/latest/format.html
<img width="881" alt="Screen Shot 2022-06-28 at 5 37 21 PM" src="https://user-images.githubusercontent.com/3595641/176315956-a1da0e3b-d865-402a-97d5-a2825db6f152.png">